docs/learn-identity.md

```diff
@@ -93,7 +93,7 @@ Copy the preimage hash. In the above image, that's
 a transaction.
 
 Next, select "Submit Proposal" and enter the previously copied preimage hash. The `locked balance`
-field needs to be at least 10 KSM. You can find out the minimum by querying the chain state under
+field needs to be at least 1.66 KSM. You can find out the minimum by querying the chain state under
```

Comment:
@DannyS03 is handling this separately in #1926

Reply:
This should be a dynamic value

Reply:
Bring upstream with @DannyS03's changes and check if it renders fine on build after tomorrow's runtime enactment without changing the defaults - it should reduce this value 50x.